indy_node/server/node.py

```diff
@@ -451,12 +358,14 @@ def executeDomainTxns(self, ppTime, reqs: List[Request], stateRoot,
         :param ppTime: the time at which PRE-PREPARE was sent
         :param req: the client REQUEST
         """
-        return self.commitAndSendReplies(self.reqHandler, ppTime, reqs,
+        req_handler = self.get_req_handler(DOMAIN_LEDGER_ID)
+        return self.commitAndSendReplies(req_handler, ppTime, reqs,
                                          stateRoot, txnRoot)
 
     def executeConfigTxns(self, ppTime, reqs: List[Request], stateRoot,
```

Comment:
Can we use generic code for excuteXTxns?

Reply:
I used more generic code in the next PR